### PR TITLE
Leave room button

### PR DESCRIPTION
**Pull request by erikjohnston**

### CODE REVIEW

File: src/controllers/organisms/RoomList.js [hunk 1]
```diff
@@ -96,7 +96,14 @@ module.exports = {
     },
 
     getRoomList: function() {
-        return RoomListSorter.mostRecentActivityFirst(MatrixClientPeg.get().getRooms());
+        return RoomListSorter.mostRecentActivityFirst(
+            MatrixClientPeg.get().getRooms().filter(function(room) {
+                return room.hasMembershipState(
+                    MatrixClientPeg.get().credentials.userId,
+                    "join"
```
Comment (Kegsay) [inline src/controllers/organisms/RoomList.js]: Does this work for invites?

**Comment by Kegsay** [comment]
LGTM other than query.

File: src/controllers/organisms/RoomList.js (hunk 1 above)
Comment (erikjohnston) [inline src/controllers/organisms/RoomList.js]: Oops

**Comment by Biep** [comment]
A button to leave the room might invite people to leave the room whenever they visit another - akin to logging of whenever they stop using Riot.

**Pull request merged**